### PR TITLE
Curvefs/tool: optimize output

### LINKED ISSUE #792: curvefs_tools: Querying the MDS status standy The MDS information fails
**Describe the bug (描述bug)**
Querying the MDS status standy The MDS information fails

root@pubbeta2-curve4:/usr/local/curvefs/etcd# curvefs_tool  status-mds
mds version: curvefs::mds::heartbeat::HeartbeatServiceImpl+curvefs::mds::MdsServiceImpl+curvefs::mds::topology::TopologyServiceImpl
leader mds: 10.182.26.16:6700
standy mds: [ ].
offline mds: [ 10.182.26.17:6700 10.182.26.18:6700 ].

**To Reproduce (复现方法)**

**Expected behavior (期望行为)**

**Versions (各种版本)**
OS:
Compiler:
curve-mds:
curve-chunkserver:
curve-snapshotcloneserver:
curve-sdk:
nebd:
curve-nbd:

**Additional context/screenshots (更多上下文/截图)**

### PR DESCRIPTION
**Pull request by Cyber-SiKu**
### What problem does this PR solve?

Issue Number: close #792 #793 #792 

Problem Summary:

### What is changed and how it works?

1. remove useless error
2. query mds status using dummy port
3. add status
4. fix version show in mds,metaserver,tool

What's Changed:

How it Works:

Side effects(Breaking backward compatibility? Performance regression?): 

### Check List

- [ ] Relevant documentation/comments is changed or added
- [ ] I acknowledge that all my contributions will be made under the project's license

### CODE REVIEW

File: curvefs/src/tools/README.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ Usage:
 curvefs_tool [Command] [OPTIONS...]
 ```
 
-When you are not sure how to use a command, --exapmle can give you an example of use:
+When you are not sure how to use a command, --exapmle can give you an example of usage:
```
Comment (SeanHai) [inline curvefs/src/tools/README.md:11]: --exapmle -> --example

File: curvefs/src/tools/README.md [hunk 2]
```diff
@@ -249,6 +249,8 @@ curvefs_tool build-topology
 
 Output:
 
+**if success, there is no output.**
```
Comment (SeanHai) [inline curvefs/src/tools/README.md:252]: Add a TODO in cpp, print some info when exec successfully.

File: curvefs/src/tools/curvefs_tool.h [hunk 3]
```diff
@@ -68,11 +68,27 @@ class CurvefsTool {
      * @details
      */
     virtual int Init() = 0;
+    /**
```
Comment (SeanHai) [inline curvefs/src/tools/curvefs_tool.h:71]: add a blank line

File: curvefs/src/tools/status/curvefs_status_base_tool.cpp [hunk 4]
```diff
@@ -73,7 +73,15 @@ void StatusBaseTool::AfterGetMetric(const std::string hostAddr,
                 errorHosts_.insert(hostAddr);
             }
         } else if (subUri == versionSubUri_) {
-            version_ = value;
+            std::string keyValue;
+            if (!metricClient_->GetKeyValueFromString(value, versionKey_,
+                                                      &keyValue)) {
+                version_ = keyValue;
+            } else {
+                std::cerr << "parse " << versionKey_ << " form " << hostAddr
```
Comment (SeanHai) [inline curvefs/src/tools/status/curvefs_status_base_tool.cpp:81]: errorOutput_ ?

File: curvefs/src/tools/README.md (hunk 1 above)
Comment (Cyber-SiKu) [reply curvefs/src/tools/README.md:11]: > --exapmle -> --example

fix

File: curvefs/src/tools/README.md (hunk 2 above)
Comment (Cyber-SiKu) [reply curvefs/src/tools/README.md:252]: > Add a TODO in cpp, print some info when exec successfully.

fix

File: curvefs/src/tools/curvefs_tool.h (hunk 3 above)
Comment (Cyber-SiKu) [reply curvefs/src/tools/curvefs_tool.h:71]: fix

File: curvefs/src/tools/umount/curvefs_umount_fs_tool.cpp [hunk 5]
```diff
@@ -93,7 +93,7 @@ void UmountFsTool::AddUpdateFlags() {
 bool UmountFsTool::AfterSendRequestToHost(const std::string& host) {
     bool ret = false;
     if (controller_->Failed()) {
-        std::cerr << "send umount fs request " << FLAGS_mountpoint
+        errorOutput_ << "send umount fs request " << FLAGS_mountpoint
```
[the remark this answers is not in the document]
Comment (Cyber-SiKu) [reply curvefs/src/tools/umount/curvefs_umount_fs_tool.cpp:96]: > align

fix

**Review by SeanHai — APPROVED** [review]

**Review by wu-hanqing — APPROVED** [review]

**Comment by Cyber-SiKu** [comment]
recheck

**Comment by Cyber-SiKu** [comment]
recheck

**Pull request merged**